plugins/woocommerce-admin/client/marketplace/components/likert-scale/likert-scale.scss

```
@@ -0,0 +1,51 @@
+.woocommerce-marketplace__likert-scale {
+	display: flex;
+	list-style: none;
+	margin: 0;
+	border: 2px solid transparent;
+
+	&.validation-failed {
+		border-color: var(--wc-red);
```

🗒️  **Right color?** I need to recheck the documentation to see if there's a preferred/approved color for indicating a validation failure.

---

I can't find an existing example of the colours. We have some reds already defined in the base styles under _colors.scss and _colors.native.scss. The alert red can be immediately used just like this:

```suggestion
		border-color: $alert-red;
```

That said, I'm not sure if that is the correct red for validation. Might be one to check with Poli.